client/express-checkout/blocks/components/express-checkout-component.js

```diff
@@ -2,11 +2,38 @@
  * External dependencies
  */
 import { ExpressCheckoutElement } from '@stripe/react-stripe-js';
+/**
+ * Internal dependencies
+ */
 import {
 	shippingAddressChangeHandler,
 	shippingRateChangeHandler,
 } from '../../event-handlers';
 import { useExpressCheckout } from '../hooks/use-express-checkout';
+import { PAYMENT_METHOD_NAME_EXPRESS_CHECKOUT_ELEMENT } from 'wcpay/checkout/constants';
+
+const getPaymentMethodsOverride = ( enabledPaymentMethod ) => {
+	const allDisabled = {
+		amazonPay: 'never',
+		applePay: 'never',
+		googlePay: 'never',
+		link: 'never',
+		paypal: 'never',
+	};
+
+	const enabledParam = [ 'applePay', 'googlePay' ].includes(
+		enabledPaymentMethod
+	)
+		? 'always'
+		: 'auto';
```

Comment:
Trying to understand this: In which scenario would we want other payment methods to be `auto`?

Reply:
`always` is only an option for ApplePay and GooglePay. [Docs](https://docs.stripe.com/js/elements_object/create_express_checkout_element#express_checkout_element_create-options-paymentMethods)

Reply:
In our current implementation it won't ever happen. I let it this way for future support of Link, Paypal or Amazon Pay.

Reply:
Got it. 👍